pkgs/development/tools/ruff/default.nix

```diff
@@ -1,33 +1,45 @@
 { lib
 , rustPlatform
 , fetchFromGitHub
+, fetchpatch
 , installShellFiles
 , stdenv
 , darwin
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "ruff";
-  version = "0.0.265";
+  version = "0.0.267";
 
   src = fetchFromGitHub {
     owner = "charliermarsh";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-b45kPvN2yILZBvP8eSNfWD6gpinwi9RsvmonRGYj0cI=";
+    hash = "sha256-0FMIjASw1DNJG0+4ERXTc6MKekOogSL5LsnEHZZNOYQ=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/i4agnywlf48spmgh9199f5nmqhsads3q-source.drv':
         specified: sha256-0FMIjASw1DNJG0+4ERXTc6MKekOogSL5LsnEHZZNOYQ=
            got:    sha256-inbW+oobW0hAsNdvJoiHvKoKAUjcuhEUrJe7fh5c6go=

Reply:
not sure how that happened, there might have been a retag

this should be fixed now